terraform/main.tf

```diff
@@ -84,10 +84,16 @@ resource "google_cloud_run_v2_job" "groundwatersqldumpjob" {
             tar -zxf oras_1.3.0_*.tar.gz -C oras-install/
             ./oras-install/oras pull ghcr.io/cgs-earth/arizona-groundwater-dump:latest
 
+            # in pygeoapi the jobs table needed for the postgresql ogc api process provider is not created by default
+            # as such we need to create it manually; we fetch the sql from the pygeoapi repo since this is the file that is used
+            # to create the jobs table in the test
+            curl --fail --show-error --silent https://raw.githubusercontent.com/geopython/pygeoapi/refs/heads/master/tests/data/postgres_manager_full_structure.backup.sql -o create_jobs_table.sql
+            psql -h /cloudsql/${google_sql_database_instance.postgis.connection_name} -U ${var.POSTGRES_USER} -d postgres -f create_jobs_table.sql
```

Comment:
I don't think you need this if you are directly connected (atleast for a postgres container). Or more I mean we arent using any of the ENV variables that we are creating for the container. We should be if we are setting the variables to https://www.postgresql.org/docs/current/libpq-envars.html?